### PR TITLE
[1LP][RFR] Fix and optimize some automate tests

### PR DESCRIPTION
**Pull request by mfalesni**
{{pytest: cfme/tests/automate/test_domain.py cfme/tests/automate/test_namespace.py cfme/tests/automate/test_class.py cfme/tests/automate/test_instance.py cfme/tests/automate/test_method.py -v --long-running }}

### CODE REVIEW

File: cfme/automate/explorer/method.py [hunk 1]
```diff
@@ -43,7 +43,8 @@ def is_displayed(self):
             self.in_explorer and
             self.title.text.startswith('Automate Method [{}'.format(
                 self.context['object'].display_name or self.context['object'].name)) and
-            self.fqdn.is_displayed and self.fqdn.text == self.context['object'].tree_path_name_only)
+            self.fqdn.is_displayed and
+            self.fqdn.text == self.context['object'].tree_path_name_only[1:])
```
Comment (mshriver) [inline cfme/automate/explorer/method.py]: **Optional**
A comment for what's sliced here

File: utils/appliance/__init__.py [hunk 2]
```diff
@@ -2512,6 +2513,9 @@ def browser(self):
     def create_view(self, view_class, o=None, override=None):
         o = o or self
         if override is not None:
-            o = OverrideWithDict(o, override)
+            new_obj = copy(o)
```
Comment (mshriver) [inline utils/appliance/__init__.py]: :+1:

**Comment by mfalesni** [comment]
@mshriver Your comment addressed and I applied a BZ blocker on the failing test due to a known issue.

**Comment by mfalesni** [comment]
The two errors in 5.6 are caused by a known bug. THerefore this PR is ready for merge.

**Pull request merged**